src/Microsoft.TestPlatform.Utilities/InferRunSettingsHelper.cs

```diff
@@ -103,6 +105,45 @@ public static void UpdateCollectSourceInformation(XPathNavigator runSettingsNavi
         }
 
         /// <summary>
+        /// Remove the <c>RunConfiguration.DesignMode</c> value for a run settings.
+        /// </summary>
+        public static void RemoveDesignMode(XPathNavigator runSettingsNavigator)
```

Comment:
DRY: Can we have generalize method here, rather than one method for each element?